packages/components/src/components/form/SelectBar/SelectBar.stories.tsx

```
@@ -42,8 +43,13 @@ const meta: Meta = {
             },
         },
         selectedOption: {
+            options: Object.values(options).map(({ value }) => value),
             control: {
-                type: 'text',
+                type: 'select',
+                labels: options.reduce(
+                    (acc, option) => ({ ...acc, [option.value]: option.label }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)